spec/praxis/routing_config_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+describe Praxis::RoutingConfig do
+
+  let(:resource_definition) do
+    Class.new do
+      include Praxis::ResourceDefinition
+      def self.name; 'MyResource'; end
+    end
+  end
+
+  let(:routing_block) { Proc.new{} }
+  let(:default_route_prefix) { "/" + resource_definition.name.split("::").last.underscore }
+
+  subject(:routing_config){ Praxis::RoutingConfig.new(&routing_block) }
+
+  its(:version) { should eq('n/a') }
+  its(:prefix ) { should eq('') }
+
+  context '#prefix' do
+    it 'sets the prefix' do
+      routing_config.prefix '/'
+      expect(routing_config.prefix).to eq('/')
+    end
+
+    it 'is additive' do
+      routing_config.prefix '/people/:id'
+      routing_config.prefix '/address'
+      expect(routing_config.prefix).to eq('/people/:id/address')
+    end
+
+    it 'strips duplicated /s' do
+      routing_config.prefix '/'
+      routing_config.prefix '/people'
+      expect(routing_config.prefix).to eq('/people')
+    end
+
+  end
```

Comment:
I think we need a context for `add_route` at least to test/show that passing a path, that starts with `//` is treated as absolute and the prefix is ignored...